### DISCUSSION TITLE
When I uncompress a zip file in my permanently messy downloads folder, the newly created folder no longers stays unsorted as the last entry

### DISCUSSION THREAD

**Discussion by shodanx2** [valinet/ExplorerPatcher#2470] [General]
It automatically gets sorted to wherever it should be if I had just pressed the refresh button.

Is there a way to prevent these refresh after unzipping a file ? 
Or force the unzipped folder to remain at the end of the list, no matter what the current sort order happens to be ?

So that it is always at the same place, instead of need to scroll to where the letter "R" goes in the current reverse alphanumeric soft, but in the folders grouping, for instance. 

Maybe even a button "enter latest created folder" in a permanent location

**Comment by pyrates999** [comment]
Is this caused by EP or just something windows explorer does on its own?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.
